### PR TITLE
Feature/support5.2

### PR DESCRIPTION
**Pull request by jimmypuckett**
Updating the plugin to work with Laravel 5.2

### CODE REVIEW

File: tests/Commands/PurgeCommandTest.php [hunk 1]
```diff
@@ -105,6 +107,39 @@ private function setUpMocks()
...
+     * which we do not own, but I really cannot come up with a better way to assert that the correct info is being
+     * sent to the console.
+     *
+     * @see https://github.com/laravel/framework/commit/2f167031ca7d9660d3524d16cbba24eae5c759d7
+     *
+     * @param $line
+     *
+     * @return array
+     */
+    private function checkVerbosity($line)
```
Comment (ssfinney) [inline tests/Commands/PurgeCommandTest.php:123]: Can we briefly talk about this one?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.yml]
+indent_style = space
+indent_size = 2
```
Comment (ssfinney) [inline .editorconfig:16]: I'm assuming this is identical to Laravel's editorconfig?

File: composer.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
         "phpunit/phpunit": "~4.0",
         "psy/psysh": "~0.5.1",
         "satooshi/php-coveralls": "~0.6.1",
-        "symfony/var-dumper": "2.7.*"
+        "symfony/var-dumper": "~2.7|3.0.*"
     },
     "autoload": {
         "psr-4": {
```
Comment (ssfinney) [inline composer.json:32]: Why did you not do a `composer update` to update the composer.lock file?

File: .editorconfig (hunk 2 above)
Comment (jimmypuckett) [inline .editorconfig:16]: Yes, exact copy.

File: composer.json (hunk 3 above)
Comment (jimmypuckett) [inline composer.json:32]: I am not sure that I follow the question. We had it locked to 2.7.*, but Laravel now supports >=2.7 & >=3, so this makes it follow Laravel.

File: tests/Commands/PurgeCommandTest.php (hunk 1 above)
Comment (jimmypuckett) [inline tests/Commands/PurgeCommandTest.php:123]: Sure, call me when you get a chance.

**Pull request merged**